package.json

```diff
@@ -15,19 +15,19 @@
     "demo-ssr": "npm run dev --workspace=playground-ssr",
     "build": "npm run build --workspace=lib",
     "test:e2e": "npm run test:e2e --workspace=lib",
-    "prepare": "husky install"
+    "prepare": "husky"
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.2.2",
-    "@commitlint/config-conventional": "^19.2.2",
-    "@typescript-eslint/parser": "^7.16.1",
-    "eslint": "^8.57.0",
+    "@commitlint/cli": "^19.5.0",
+    "@commitlint/config-conventional": "^19.5.0",
+    "@typescript-eslint/parser": "^8.12.1",
+    "eslint": "^9.13.0",
```

Comment:
@manuelschroederdev we shouldn't update to v9 of eslint until we get the new pnpm setup with eslint flat config and `@storyblok/eslint-config` package